### PR TITLE
 [TH2-3210]  th2-infra > add ingress, configmaps and secrets custom annotations

### PR DESCRIPTION
**Pull request by Maximus905**

### CODE REVIEW

File: th2-service/templates/_infra-mgr-service.yaml [hunk 1]
```diff
@@ -6,9 +6,11 @@ apiVersion: v1
 kind: Service
 metadata:
   name: infra-mgr
+  annotations:
+  {{- include "common-annotaitions" . | nindent 4  }}
```
Comment (bop0hz) [inline th2-service/templates/_infra-mgr-service.yaml:10]: typos everywhere annotaitions

File: th2-service/templates/ingress.yaml [hunk 2]
```diff
@@ -3,15 +3,7 @@ apiVersion: networking.k8s.io/v1beta1
...
-    nginx.ingress.kubernetes.io/use-regex: "true"
-    nginx.ingress.kubernetes.io/configuration-snippet: |
-      if ($request_uri ~ "^/rabbitmq(/.*)") {
-        proxy_pass http://upstream_balancer$1;
-        break;
-      }
+    {{- include "ingress-annotaitions" . | nindent 4  }}
   name: ingress-service
 spec:
   rules:
```
Comment (bop0hz) [inline th2-service/templates/ingress.yaml:9]: remove rabbitmq path

**Review by bop0hz — CHANGES_REQUESTED** [review]
please fix

File: th2-service/templates/_infra-mgr-service.yaml (hunk 1 above)
Comment (Maximus905) [reply th2-service/templates/_infra-mgr-service.yaml:10]: fixed

File: th2-service/templates/ingress.yaml (hunk 2 above)
Comment (Maximus905) [reply th2-service/templates/ingress.yaml:9]: rabbitmq path was moved into rabbit values

File: th2-service/values.yaml [hunk 3]
```diff
@@ -40,21 +48,24 @@ infraEditor:
...
+      ref: release-v0.6.1
+      path: ./
       # external mirror
       # repository: https://th2-net.github.io
-      repository: http://infra-repo:8080
-      version: 0.6.0
-      name: infra-operator-tpl
+      # repository: http://infra-repo:8080
+      # version: 0.6.1
+      # name: infra-operator-tpl
```
Comment (bop0hz) [inline th2-service/values.yaml:68]: fix before merge

**Review by bop0hz — APPROVED** [review]

**Pull request merged**